package/rpm/HIRS.spec

```diff
@@ -36,11 +36,11 @@ Summary         : Host Integrity at Runtime and Startup (HIRS) Provisioner
 Group           : System Environment/Base
 
 %if 0%{?build6}
-Requires        : tpm_module, java-1.8.0, wget, util-linux, chkconfig, sed, initscripts, coreutils, dmidecode, lshw, bash%{?RPM_EXTRA_CLIENT_DEPENDENCIES}
+Requires        : tpm_module, java-1.8.0, wget, util-linux, chkconfig, sed, initscripts, coreutils, dmidecode, paccor, bash%{?RPM_EXTRA_CLIENT_DEPENDENCIES}
```

Comment:
This dependency will be fine if you build Paccor from source and install it, correct?

Reply:
No. If you install something outside of the realm of the package manager (in this case RPM), the package management will not know about it, so dependencies cannot be confirmed. However, the installer can do an install that ignores dependencies.

Reply:
So if that's the case do we want to list paccor as a requires? It's not currently listed in any package repos and would then force the user to always do an install and ignore dependencies, which seems suspect.

Reply:
By install, I thought you meant "make install" on source code. If the user installs PACCOR via RPM, everything is fine. Yes, we definitely want to list PACCOR as a dependency because the Provisioners don't work at all without it.

Reply:
Well I did mean `make install` but if they can just install the RPM and that registers it with the Package Manager such that it won't block the installation of the Provisioner, then this is fine with me. As a final check, that's what you're suggesting?

Reply:
Yes.